.github/workflows/checks.yaml

```diff
@@ -24,6 +24,33 @@ jobs:
       - uses: amannn/action-semantic-pull-request@e9fabac35e210fea40ca5b14c0da95a099eff26f
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Types include:
+          #   - fix: fixes
+          #   - feat: features and enhancements
+          #   - chore: non-feature or enhancement (i.e. docs, ci, linting, automated, etc)
+          types: |
+            fix
+            feat
+            chore
```

Comment:
```suggestion
            chore
            refactor
            revert
```

Reply:
@jrschumacher Should we also keep the `test` type. Referencing the default types here. 

https://github.com/commitizen/conventional-commit-types/blob/master/index.json